neil

```diff
@@ -270,15 +270,21 @@
         edn-nodes (edn-nodes edn-string)
         lib (:lib opts)
         lib (symbol lib)
-        git? (or (:sha opts)
-                 (:latest-sha opts))
+        explicit-git? (or (:sha opts)
+                          (:latest-sha opts))
```

Comment:
Also not super happy with the `explicit-git?` name. Selected? User-selected? `user-wants-git?`? (nitpick)